content/js1/sprints/4/day-plan/index.md

```diff
@@ -6,16 +6,17 @@ menu_level = ['sprint']
 weight = 3
 backlog= 'Module-JS1'
 backlog_filter= 'Week 4'
-
 [[blocks]]
 name="energiser"
 [[blocks]]
-name="pd-placeholder"
+name= "Asking good technical questions"
+src= "https://cyf-pd.netlify.app/blocks/asking-good-technical-questions/readme/"
 [[blocks]]
 name="morning-break"
 [[blocks]]
-name="Handling undefined"
-src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/undefined"
+name="⌚ Time string"
+src="https://github.com/CodeYourFuture/CYF-Workshops/blob/main/time-string/readme.md"
```

Comment:
```suggestion
src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/time-string"
```

Reply:
Try that? The build is failing on this call

https://app.netlify.com/sites/cyf-curriculum/deploys/65031ed10d8ecd000841dab7

Reply:
just removed the filename from the path